internal/nodegroups/nodegroups.go

```diff
@@ -18,6 +18,7 @@ package nodegroups
 
```

Comment:
why do we have this and https://github.com/openshift-kni/numaresources-operator/blob/main/api/v1/helper/nodegroup/nodegroup.go
the function GetPoolNamesFrom is similar to https://github.com/openshift-kni/numaresources-operator/blob/main/api/v1/helper/nodegroup/nodegroup.go#L136 should we have one internal helper?

Reply:
yes, we should merge in a future PR.